include/PianoRoll.h

```diff
@@ -209,6 +209,7 @@ protected slots:
 	void noteLengthChanged();
 	void keyChanged();
 	void quantizeNotes(QuantizeActions mode = QuantizeBoth);
+	void strumNotesDn();
```

Comment:
At the moment this function strums like this /  ... I would call that up. And down like this \

The best thing would be if the function was defined like:

    void strumNotes(bool down);

or something like that. It would make it possible to strum up and down using the same function, like:

    strumNotes(true);  // down
    strumNotes(false);  // up

Instead of making another function for `strumNotesUp()`

Reply:
Up/Down
I saw it as from where the change _came_. In strumNotesDn() we start low and end higher.
In strumNotesUp() we start UP and end lower
That was the logic, I will add a headsup for that.